### PR TITLE
Show books list, add and remove book functionality working

### PR DESCRIPTION
**Pull request by Oscar-Espinoza**
In this request, we added the functionalities to add and remove books from the books list

### CODE REVIEW

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Invalid Code Review Request ❌

Hi @Oscar-Espinoza, This is @sja-thedude your code reviewer.

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that doesn't have linters passing and also there are no commits from your partner.
Note: It is important that both contribute in order for you to proceed and you cant disable linters, you have to fix them.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: app.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  localStorage.setItem('books', JSON.stringify(initialBooks));
+}
+
+let booksArr = JSON.parse(localStorage.getItem('books'));
+
+const remove = (removedBook) => {
+  booksArr = booksArr.filter((book) => book.id !== removedBook.id);
+  localStorage.setItem('books', JSON.stringify(booksArr));
+  booksListEl.removeChild(removedBook);
+};
```
Comment (julie-ify) [inline app.js:25]: - [ ] The books are not being correctly removed from the local storage, it is only removed from the page. Kindly fix this. As you can see I removed the book titled "my own book" yet it exists in the local storage, and when I refresh the page it comes back. Kindly fix that.

  idod

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @Oscar-Espinoza,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by t-yanick — APPROVED** [review]
Hi @Oscar-Espinoza , 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  You implemented the changes proposed by the previous reviewer.


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**